src/components/common/Picker/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
```

Comment:
This one is not used, do we need to import it anyways?

Reply:
It's needed for every React component, this will compile into something like `React.createElement(element, ...)`